releases/v19.2.0-beta.20190826.md

```diff
@@ -0,0 +1,243 @@
+- Added `\d`, `\dt`, `\du`, `\l` metacommands to the [cockroach sql](../v19.2/use-the-built-in-sql-client.html) interactive shell. [#39141][#39141] {% comment %}doc{% endcomment %}
+- Added default cluster locality topologies for multi-node [`cockroach demo`](../v19.2/cockroach-demo.html) clusters. [#40221][#40221]
+- Errors in [`cockroach demo`](../v19.2/cockroach-demo.html) are no longer logged to the command-line interface. [#40196][#40196]
+
+### Admin UI changes
+
+- Metrics about p90 and p99 percentiles of per-application average transaction time are now exposed in the [Admin UI](../v19.2/admin-ui-access-and-navigate.html). [#39429][#39429]
+
+### Bug fixes
+
+- Fixed a potential infinite loop in queries involving reverse scans. [#39084][#39084]
+- Upgraded the `grpc` library to fix connection state management bug. [#39041][#39041]
+- Properly initialize proposal quota tracking to prevent quota leak which can hang imports or other AddSSTable operations. [#39135][#39135]
+- `crdb_internal.ranges` can now be used inside [views](../v19.2/views.html). Note that such views can become invalid in future releases if `crdb_internal.ranges` changes. [#39195][#39195]
+- Prevented panic due to recording into finished tracing spans caused by acknowledging an earlier failed re-proposal when a later re-proposal exists. [#39203][#39203]
+- CockroachDB now properly computes the default port number if an IPv6 address is provided without port number with `--locality-advertise-addr`. [#39334][#39334]
+- Unary negatives in constant arithmetic expressions are no longer ignored. [#39245][#39245]
+- `pg_catalog` now shows correct type lengths for [`INT`](../v19.2/int.html) and [`FLOAT`](../v19.2/float.html). [#39337][#39337]
+- Fixed a rare panic (message: `"raft group deleted"`) that could occur during splits. [#39571][#39571]
+- Fixed a panic due to incorrect type inference of some `ARRAY(...)` expressions. [#39469][#39469]
+- [Zone configurations](../v19.2/configure-replication-zones.html) now propagate to non-gossiped system tables. [#39638][#39638]
+- [`cockroach dump`](../v19.2/sql-dump.html) now works properly when handling [`INTERLEAVED`](../v19.2/interleave-in-parent.html) tables, printing them outside of [`CREATE TABLE`](../v19.2/create-table.html) statements. [#39486][#39486]
+- Fixed internal errors generated during execution of some complicated cases of [correlated subqueries](../v19.2/subqueries.html#correlated-subqueries). [#39818][#39818]
+- Using `ORDER BY` inside aggregation functions is now fully supported. [#40242][#40242]
+- Replicas can now be moved between stores without entering an intermediate configuration that violates the [zone constraints](../v19.2/configure-replication-zones.html). Violations may still occur during zone config changes, [decommissioning](../v19.2/remove-nodes.html), and in the presence of dead nodes. [#40284][#40284]
```

Comment:
Discussed this with @tbg. This isn't on by default yet, so we should remove from the notes for now.